Userland/Libraries/LibGL/SoftwareRasterizer.cpp

```diff
@@ -34,7 +34,7 @@ constexpr static T mix(const T& x, const T& y, float interp)
     return x * (1 - interp) + y * interp;
 }
 
-static Gfx::RGBA32 to_rgba32(const FloatVector4& v)
+constexpr static Gfx::RGBA32 to_rgba32(const FloatVector4& v)
```

Comment:
Since the goal here is to make sure the function gets inlined, let's use `ALWAYS_INLINE` to communicate intent.

Reply:
Done!